x/smart-account/specs/README.md

```
@@ -0,0 +1,147 @@
+
+**This logic is now programmable via smart contracts & additional configurations,vastly expanding how transactions can be implemented.**
+
+
+### Minimum Requirements
+- One secp256k1 keypair to register to the smart-account, funded with gas to pay for transsaction fees
+- An authentication method of choice
+- Optional - an `TxExtension` appended to each message, in order to specific which authenticator to use (defualts to regular key ecdsa authorization)
+
+## Workflow Module Uses To Authorize Transactions
+
+###  1. Inital Gas Limit
```

markdownlint-cli2 (0.17.2)

Multiple spaces after hash on atx style heading
null

(MD019, no-multiple-space-atx)